src/jsglue.cpp

```diff
@@ -1062,13 +1087,13 @@ CallUnbarrieredObjectTracer(JSTracer* trc, JSObject** objp, const char* name)
 void
 CallObjectRootTracer(JSTracer* trc, JSObject** objp, const char* name)
 {
-    JS::UnsafeTraceRoot(trc, objp, name);
+    JS::TraceRoot(trc, objp, name);
```

Comment:
This method was renamed at https://bugzilla.mozilla.org/show_bug.cgi?id=1744059.